firstvoices/backend/serializers/site_serializers.py

```diff
@@ -50,6 +52,12 @@ def get_default_menu(self):
         default_menu = AppJson.objects.filter(key="default_site_menu")
         return default_menu[0].json if len(default_menu) > 0 else None
 
+    def get_characters(self, site):
+        return self.get_site_content_link(site, "api:characters-list")
+
+    def get_ignored_characters(self, site):
```

Comment:
I thought ignored characters would work best as their own endpoint to avoid confusion on what is/is not an ignored character, I am open to suggestions to change how ignored characters behave in the API though

Reply:
@gmcauliffe I think games is the only use case for retrieving ignored characters right now. Do you have a preference about how to retrieve them?